src/Features/Core/Portable/Microsoft.CodeAnalysis.Features.csproj

```diff
@@ -113,6 +113,7 @@
     <RestrictedInternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServerClient.Razor" Partner="Razor" Key="$(RazorKey)" />
     <RestrictedInternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServerClient.Razor.Test" Partner="Razor" Key="$(RazorKey)" />
     <InternalsVisibleTo Include="IdeCoreBenchmarks" />
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.AspNetCore" />
```

Comment:
Same here, should be `RestrictedInternalsVisibleTo` similar to the one two lines above.

Reply:
sounds good!

Reply:
hrmm... i couldn't get thsi to work.  are you sure that's the right pattern here?  i'm creating a new EA lib that *should* have IVT to our libs.  but it's that lib htat we can then audit to make sure it is not further exposing things outwards that it shoudln't.

Reply:
➡️ `RestrictedInternalsVisibleTo` is only applicable to cases where we cannot create a new assembly. (marking unresolved just for visibility)